From d55ab5e5cfe21c7563b582365a111faf2f427192 Mon Sep 17 00:00:00 2001
From: Stefan Schmidt <s.schmidt@samsung.com>
Date: Wed, 10 Sep 2014 16:29:22 +0200
Subject: [PATCH 4/4] edje_cc: Remove no longer needed extern tmp_dir.

The last two user have been switched to eina_file_mkstemp so we don't
need to setup tmp_dir anymore.
---
 src/bin/edje/edje_cc.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/bin/edje/edje_cc.h b/src/bin/edje/edje_cc.h
index da7e9d5..6a6d79f 100644
--- a/src/bin/edje/edje_cc.h
+++ b/src/bin/edje/edje_cc.h
@@ -253,7 +253,6 @@ extern Eina_List             *snd_dirs;
 extern Eina_List             *vibration_dirs;
 extern Eina_List             *data_dirs;
 extern char                  *file_in;
-extern char                  *tmp_dir;
 extern char                  *file_out;
 extern char                  *watchfile;
 extern char                  *license;
-- 
1.8.4

